examples/producer.go

```diff
@@ -23,9 +23,9 @@ func main() {
 	config.Net.TLS.Enable = true
 	config.Net.TLS.Config = nil
 
-	superstream.Init("token", config)
+	newConfig := superstream.Init("token", config)
```

Comment:
can we change it to be the same config?
config = init()...
and add the consumerGroup and servers so the user will be aware of that